pyquil/api/_wavefunction_simulator.py

```diff
@@ -247,50 +270,39 @@ def run_and_measure_payload(
     if not isinstance(trials, int):
         raise TypeError("trials must be an integer")
 
-    payload = {
-        "type": TYPE_MULTISHOT_MEASURE,
-        "qubits": list(qubits),
-        "trials": trials,
-        "compiled-quil": quil_program.out(calibrations=False),
-    }
-
-    if random_seed is not None:
-        payload["rng-seed"] = random_seed
-
-    return payload
+    return RunAndMeasureProgramRequest(
+        program=quil_program.out(calibrations=False),
+        qubits=list(qubits),
+        trials=trials,
+        measurement_noise=None,
```

Comment:
The old payloads didn't set any noise config for this and for `wavefunction_request/payload` below